patches/sku@11.4.5.patch

```diff
@@ -14,4 +14,4 @@ index 0029c6c8074ae05a91b2d11fa4831fe878206c71..be6b7f1955c8b97a713c036f3815af5b
 +      // validatePeersDeps();
```

Comment:
I don't think this patch works. When I did the patch for 11.4.5 the diff markers were different.

Reply:
Works for me with a clean install. 🤔